### PR TITLE
Update touch ID enrolment setting

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
This PR depends on https://github.com/appium/appium-ios-simulator/pull/126 and https://github.com/appium/appium-base-driver/pull/130

The idea is to get rid of separate _allowTouchIdEnroll_ capability, simplify the flow and switch enrolment state using UI AppleScript. This is more reliable, but requires Appium process to have system Accessibility access.

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
@imurchie We cannot, because Appium process does not have permissions to Accessibility on Travis. Our functional tests will always fail.

**Comment by imurchie** [comment]
Then they should only be skipped on Travis, not removed altogether. We want to be able to test locally.

File: test/functional/basic/touch-id-e2e-specs.js [hunk 1]
```diff
@@ -32,29 +32,26 @@ if (!process.env.REAL_DEVICE) {
...
     describe('touchID enrollment functional tests applied to TouchId sample app', function () {
-      beforeEach(async () => {
-        caps = Object.assign(TOUCHIDAPP_CAPS);
-        caps.allowTouchIdEnroll = true;
-        driver = await initSession(caps);
+      beforeEach(async function () {
+        if (process.env.TRAVIS) {
+          // ignore tests on Travis, since Appium process does not have access to
+          // system Accessibility there
+          this.skip();
```
Comment (imurchie) [inline test/functional/basic/touch-id-e2e-specs.js]: This needs a `return`, or the function continues. No need to create the session and all.

Comment (mykola-mokhnach) [inline test/functional/basic/touch-id-e2e-specs.js]: added return

**Pull request merged**